### PR TITLE
Update ruby pages.

### PR DESCRIPTION
**Pull request by jackorp**
@pvalena  JFYI

### CODE REVIEW

**Comment by jackorp** [comment]
Fixes : https://github.com/developer-portal/content/issues/472

File: tech/languages/ruby/ruby-installation.md [hunk 1]
```diff
@@ -61,48 +49,46 @@ RUBYPICK=_jruby_
...
-source ~/.bashrc
-exec $SHELL
-```
-
-Install Ruby
 
+Next step we have to refresh the shell to make the binaries available. This can be achieved
+by either the following command or opening a new terminal:
+```console
+$ exec $SHELL
```
Comment (pvalena) [inline tech/languages/ruby/ruby-installation.md:68]: I do not think this will work without the sourcing of bashrc, but feel free to try.

**Review by pvalena — APPROVED** [review]
LGTM, thanks!

File: tech/languages/ruby/ruby-installation.md (hunk 1 above)
Comment (jackorp) [reply tech/languages/ruby/ruby-installation.md:68]: It seems to just work. However, thanks to retrying this I found 2 things: Perl is required and there is rbenv RPM package in Fedora.

File: tech/languages/ruby/ruby-installation.md [hunk 2]
```diff
@@ -61,48 +49,46 @@ RUBYPICK=_jruby_
 
 The first step is to install dependencies for Ruby.
 
-```
-sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
+```console
+$ sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
```
Comment (jackorp) [inline tech/languages/ruby/ruby-installation.md:53]:
```suggestion
$ sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel perl-FindBin
```

File: tech/languages/ruby/ruby-installation.md (hunk 1 above)
Comment (jackorp) [reply tech/languages/ruby/ruby-installation.md:68]: It works.

**Pull request merged**